app/styles/modules/_custom-rows.scss

```diff
@@ -162,6 +162,24 @@ ul.links {
   margin: 0 10px;
 }
 
+.show-password-label {
+  background-color: $show-password-background-color;
+  color: $show-password-label-color;
+  -webkit-transition: background-color 450ms,
+                      color 450ms;
+  transition: background-color 450ms,
```

Comment:
@TDA autoprefixer might do this for us already where you only need to set `transition` once.